### PR TITLE
[integration] Remove __RCSID__

### PR DESCRIPTION
**Pull request by chrisburr**
Littering unrelated commits with the removal `__RCSID__ ` makes PRs harder to review for no good reason so I propose we do it in one go and be done with it.

This PR is simply replacing `\n+__RCSID__ = "\$Id\$"\n+` with `\n` and letting pre-commit fix the whitespace.

### CODE REVIEW

**Comment by TaykYoku** [comment]
maybe also remove the following, in the same way? to review it once together
```python
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
```

**Comment by chrisburr** [comment]
As soon as this is merged I'm going to make that PR.

Though that one is a little more complicated as we do need to keep Python 2 support for some files which are used by the site directors to send pilots.

**Review by TaykYoku — APPROVED** [review]
Maybe the same need to do with:
```python
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
```

File: src/DIRAC/DataManagementSystem/Client/DataManager.py [hunk 1]
```diff
@@ -38,9 +38,6 @@
 from DIRAC.ResourceStatusSystem.Client.ResourceStatus import ResourceStatus
 
 # # RSCID
```
[the remark this answers is not in the document]
Comment (chrisburr) [reply src/DIRAC/DataManagementSystem/Client/DataManager.py:40]: There are also some buggy entries like `__RCSID__ = "$Id $"` to clean up after this is merged.

**Pull request merged**